fetch/cosmwasm_contract/src/contract.rs

```diff
@@ -39,7 +39,11 @@ pub fn init<S: Storage, A: Api, Q: Querier>(
 
     let contract_addr_human = deps.api.human_address(&env.contract.address)?;
 
-    if msg.lower_swap_limit > msg.upper_swap_limit || msg.lower_swap_limit <= msg.swap_fee {
+    if msg.reverse_swap_min > msg.reverse_swap_max || msg.reverse_swap_min <= msg.reverse_swap_fee {
+        return Err(StdError::generic_err(ERR_SWAP_LIMITS_INCONSISTENT));
```

Comment:
This woukld perhaps deserve dedicated error code for reverse swap limits (to distinguish from error for (forward) swap limits).

Reply:
yes I can.
I tried to do the minimum changes to contract to get the desired behavior

Reply:
done